glide.yaml

```diff
@@ -1,30 +1,30 @@
 package: github.com/openshift/cluster-openshift-apiserver-operator
 import:
 - package: k8s.io/apimachinery
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/api
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/apiserver
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/client-go
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/kube-aggregator
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/code-generator
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/apiextensions-apiserver
   repo: https://github.com/openshift/kubernetes-apiextensions-apiserver
-  version: origin-4.0-kubernetes-1.12.4 # for nullable PR, instead of kubernetes-1.12.5
+  version: origin-4.1-kubernetes-1.13.4 # pinned to get nullable patch
 - package: sigs.k8s.io/controller-tools
   repo: https://github.com/openshift/kubernetes-sigs-controller-tools
-  version: master
+  version: origin-4.0-kubernetes-1.12.4 # TODO: move to 1.13
```

Comment:
origin-4.1-kubernetes-1.13.4

Reply:
it was failing on the PROJECT thing so we moved this back to 1.12